analyzer/src/funTest/assets/projects/synthetic/gradle-composite/project1/.gitattributes

```diff
@@ -0,0 +1,9 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Linux start script should use lf
+/gradlew        text eol=lf
+
+# These are Windows script files and should use crlf
+*.bat           text eol=crlf
+
```

Comment:
This is already in the root .gitattributes, I think it should not be required here.